test/acceptance/behave/components/ngsiv2/attributes/remove_a_single_attribute.feature

```diff
@@ -0,0 +1,537 @@
+      | /serv1/serv2/serv3/serv4/serv5/serv6/serv7/serv8/serv9/serv10 |
+      | max length allowed                                            |
+      | max length allowed and ten levels                             |
+
+  @service_path_delete_without
+  Scenario: Delete an attribute by entity ID using NGSI v2 without service path header
+    Given a definition of headers
+      | parameter      | value                    |
+      | Fiware-Service | test_delete_service_path |
+      | Content-Type   | application/json         |
+    And create "1" entities with "3" attributes
+      | parameter        | value       |
+      | entities_type    | house       |
+      | entities_id      | room        |
+      | attributes_name  | temperature |
+      | attributes_value | 34          |
+      | attributes_type  | celcius     |
+      | metadatas_number | 2           |
+      | metadatas_name   | very_hot    |
+      | metadatas_type   | alarm       |
+      | metadatas_value  | hot         |
+    And verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+    When delete an attribute "temperature_0" in entities with id "room"
+    Then verify that receive several "No Content" http code
+    And verify that the attribute is deleted into mongo
+
+  @service_path_delete_error
+  Scenario Outline: Try to delete an attribute by entity ID using NGSI v2 with wrong service header path values
+    Given a definition of headers
+      | parameter          | value                          |
+      | Fiware-Service     | test_delete_service_path_error |
+      | Fiware-ServicePath | <service_path>                 |
+      | Content-Type       | application/json               |
+    When delete an attribute "temperature_0" in entities with id "room"
+    Then verify that receive several "Bad Request" http code
+    And verify several error responses
+      | parameter   | value                                                    |
+      | error       | BadRequest                                               |
+      | description | a component of ServicePath contains an illegal character |
+    Examples:
+      | service_path |
+      | /service.sr  |
+      | /service;sr  |
+      | /service=sr  |
+      | /Service-sr  |
+      | /serv<45>    |
+      | /serv(45)    |
+
+  @service_path_delete_error
+  Scenario Outline: Try to delete an attribute by entity ID using NGSI v2 with wrong service path header values
+    Given a definition of headers
+      | parameter          | value                          |
+      | Fiware-Service     | test_delete_service_path_error |
+      | Fiware-ServicePath | <service_path>                 |
+      | Content-Type       | application/json               |
+    When delete an attribute "temperature_0" in entities with id "room"
+    Then verify that receive several "Bad Request" http code
+    And verify several error responses
```

Comment:
You say "several", but actually the response is only one (composed of error and description subfields)